CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
   ## v8.5.0
 
+  * **Added updated configuration options for transaction events and deprecate previous configs**
+    This release deprecates and replaces the following configuration options
```

Comment:
```suggestion
    This release deprecates and replaces the following configuration options:
```

Reply:
That's ok! No need to go back and change it now. We usually review the Changelog for little things like that before a release anyway. 😄